llvm/test/CodeGen/NVPTX/kernel-param-align.ll

```diff
@@ -0,0 +1,48 @@
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_60 | FileCheck %s
+; RUN: %if ptxas %{ llc < %s -march=nvptx64 -mcpu=sm_60 | %ptxas-verify %}
+
+%struct.Large = type { [16 x double] }
+
+; CHECK-LABEL: .entry func_align(
+; CHECK: .param .u64 .ptr .global .align 16 func_align_param_0
+; CHECK: .param .u64 .ptr .global .align 16 func_align_param_1
+; CHECK: .param .u64 .ptr .global .align 16 func_align_param_2
+; CHECK: .param .u64 .ptr .shared .align 16 func_align_param_3
+; CHECK: .param .u64 .ptr .const  .align 16 func_align_param_4
+define void @func_align(ptr nocapture readonly align 16 %input,
+                        ptr nocapture align 16 %out,
+                        ptr addrspace(1) align 16 %global,
+                        ptr addrspace(3) align 16 %shared,
+                        ptr addrspace(4) align 16 %const) {
+entry:
+  %0 = addrspacecast ptr %out to ptr addrspace(1)
```

Comment:
Can the bodies of these functions be completely removed?

Reply:
Done.